README.md

```diff
@@ -973,6 +998,40 @@ and run:
 $ cargo run --release --bin server
 ```
 
+At first, we are going to test our server using [Insomnia][insomnia].
+Insomnia is a free API testing tool, and it supports testing gRPC services.
```

Comment:
For the crowd interested in this blog post, I would suggest we make sure to refer to it as OSS:

```suggestion
Insomnia is an open source API testing tool, and it supports testing gRPC services.
```